src/helps_bots/missing_refs.php

```
@@ -0,0 +1,85 @@
+        $name = $cite["name"];
+        // ---
+        $rr = $fulls[$name] ?? false;
+        if (!$rr) {
+            $empty_refs[$name] = $cite;
+        }
+    }
+    // ---
+    return $empty_refs;
+}
+
+function fix_missing_refs($text, $sourcetitle, $mdwiki_revid)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$sourcetitle'. (undefined)

(UnusedFormalParameter)